File: 30 Days of Code/Day08: Dictionaries and Maps/dictionaries_and_maps.rb
n = gets.chomp.to_i
hash = {}

(1..n).each do |i|
  input = gets.chomp.split(' ')
  hash["#{input[0]}"] = input[1]
end

STDIN.each_line do |query|
  if hash[query.chomp]
    print "#{query.chomp}=#{hash[query.chomp]}\n"
  else
    print "Not found\n"
  end
end

File: 30 Days of Code/Day08: Dictionaries and Maps/dictionaries_and_maps.swift
var n = Int(readLine()!)!
var phoneBook = [String: String]()

for i in 0...n - 1 {
    var input = readLine()!.characters.split(separator: " ").map(String.init)
    phoneBook[input[0]] = input[1]
}

var query = readLine()
while query != nil {
    if let number = phoneBook[query!] {
        print("\(query!)=\(phoneBook[query!]!)")
    }
    else {
        print("Not found")
    }
    query = readLine()
}
